bftengine/src/bftengine/messages/MessageBase.cpp

```diff
@@ -46,11 +46,66 @@ void MessageBase::printLiveMessages() {
 namespace bftEngine {
 namespace impl {
 
+// static class members for diagnostics server
+std::array<std::atomic<size_t>, MsgCode::LastMsgCodeVal> MessageBase::Statistics::AliveIncomingExtrnMsgsBufs{};
+static_assert(MsgCode::LastMsgCodeVal < 2000,
```

Comment:
where 2000 comes from?

Reply:
This is an approximation @hshomroni  decided to do after my comment - since he is using some std::array of size MsgCode::LastMsgCodeVal, he must be sure the array wont way too big if someone decides to jump the message codes into a very big number.